apps/Pomodoro.py

```diff
@@ -0,0 +1,304 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+# Copyright (C) 2022 github user thiswillbeyourgithub
+"""Pomodoro Application
+~~~~~~~~~~~~~~~~~~~~~~~
+
+A pomodoro app, forked from timer.py. Swipe laterally to load presets and vertically
+to change number of vibration. Vibration pattern are randomized if vibrating
+more than 4 times.
+
+    .. figure:: res/PomodApp.png
+        :width: 179
+
+        Screenshot of the Pomodoro Application
+
+"""
+
+import wasp
+import fonts
+import widgets
+import math
+import random
+from micropython import const
+
+# 2-bit RLE, 96x64, generated from png provided by plan5, 239 bytes
```

Comment:
The png is missing (other encoded icons reference the source png). Also it'd be best if this had proper attribution in the form of Co-Authored-By in the commit message

Reply:
Unfortunately I don't think I have the original icon png anymore, just the screenshot of the running app.

Reply:
A screenshot should be fine as long as it encodes to the same RLE